microsoft-edge/extensions-chromium/developer-guide/native-messaging.md

```diff
@@ -181,21 +181,22 @@ If Microsoft Edge finds the registry key at any of the previously listed locatio
 The search order for the registry locations is:
 
 ```output
-HKEY_CURRENT_USER\SOFTWARE\WOW6432Node\Microsoft\Edge\NativeMessagingHosts\
-HKEY_CURRENT_USER\SOFTWARE\WOW6432Node\Chromium\NativeMessagingHosts\
-HKEY_CURRENT_USER\SOFTWARE\WOW6432Node\Google\Chrome\NativeMessagingHosts\
 HKEY_CURRENT_USER\SOFTWARE\Microsoft\Edge\NativeMessagingHosts\
 HKEY_CURRENT_USER\SOFTWARE\Chromium\NativeMessagingHosts\
 HKEY_CURRENT_USER\SOFTWARE\Google\Chrome\NativeMessagingHosts\
 
 HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Microsoft\Edge\NativeMessagingHosts\
 HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Chromium\NativeMessagingHosts\
 HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Google\Chrome\NativeMessagingHosts\
+
 HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Edge\NativeMessagingHosts\
 HKEY_LOCAL_MACHINE\SOFTWARE\Chromium\NativeMessagingHosts\
 HKEY_LOCAL_MACHINE\SOFTWARE\Google\Chrome\NativeMessagingHosts\
 ```
 
+> [!NOTE]
+> The `HKEY_CURRENT_USER\SOFTWARE\WOW6432Node` registry node is not searched on 64-bit machines due to the way registry works on them. For more information, see [Registry Keys Affected by Windows Installations That Include Windows on Windows (WOW) Support For Multiple Processor Architectures](https://learn.microsoft.com/windows/win32/winprog64/shared-registry-keys).
+
```

Comment:
there will now be 2 Notes.  Notes are a kind of Alert - a deprecated kind of alert.
https://review.learn.microsoft.com/en-us/help/platform/markdown-reference?toc=%2Fhelp%2Fplatform%2Ftoc.json&bc=%2Fhelp%2Fplatform%2Fbreadcrumb%2Ftoc.json&branch=main#alerts-note-tip-important-caution-warning
We might keep as-is, or if i have time, I might enter a comment to reformat the notes as eg h4 headings, which gives a nice label.  I plan to do that.